macos/FluentUI/Button/Button.swift

```diff
@@ -139,6 +146,32 @@ open class Button: NSButton {
 			}
 		}
 	}
+	
+	/// the current state of the button
+	private var buttonState: ButtonState? {
```

Comment:
Does the variable need to be optional? (**ButtonState?** instead of just **ButtonState**)
You could make it non-optional and assign a default value.

This would solve the explicitly unwrapping issues throughout the file (**buttonsState!**).

Reply:
It doesn't have to be optional. Will be happy to always have it set as .normal as default

Reply:
small nit: I think "rest" is the more aligned name than normal